testutil/testnode/node_interaction_api.go

```diff
@@ -141,3 +144,73 @@ func (c *Context) PostData(account string, ns, msg []byte) (*sdk.TxResponse, err
 
 	return res, nil
 }
+
+// FillBlock will create and submit enough PayForData txs to fill a block to a
+// specified square size. It uses a crude mechanism to estimate the number of
+// PayForData txs to submit by creating creating squareSize - 1 messages that
+// each take up a single row.
+func FillBlock(cctx client.Context, squareSize int, accounts []string) ([]*sdk.TxResponse, error) {
+	msgCount := squareSize - 1
+	msgSize := squareSize*appconsts.SparseShareContentSize - 100
```

Comment:
this doesn't work as expected yet. We also need a new const for `FirstSparseShareContentSize` and rename `SparseShareContentSize` to `ContinuationSparseShareContentSize`

Reply:
> We also need a new const for FirstSparseShareContentSize and rename SparseShareContentSize to ContinuationSparseShareContentSize

Disregard because the varint encoded into the first sparse share isn't padded to 4 bytes so we don't know the size of `FirstSparseShareContentSize`